src/hw/installApp.js

```diff
@@ -20,6 +22,19 @@ export default function installApp(
   }).pipe(
     filter(e => e.type === "bulk-progress"), // only bulk progress interests the UI
     throttleTime(100), // throttle to only emit 10 event/s max, to not spam the UI
-    map(e => ({ progress: e.progress })) // extract a stream of progress percentage
+    map(e => ({ progress: e.progress })), // extract a stream of progress percentage
+    catchError((e: Error) => {
+      if (!e || !e.message) return throwError(e);
+      const status = e.message.slice(e.message.length - 4);
+      if (status === "6a83") {
```

Comment:
it would be great if we had constants with the meaning of the apdus instead of codes somehow -.-

Reply:
yeah, we in fact have with `TransportStatusError` but it turns out this error directly comes in a string from the HSM and in fact not the device (that proxy the apdu). we'll see later what we can do)